### DISCUSSION TITLE
CallDetails

### DISCUSSION THREAD

**Discussion by venkatesh-reddi** [venkatesh-reddi/OverAlt#17] [General]
@Sakina-n72 you add **getteres** and **setters** for all the attributes you defined. And also **add an empty constructor**. 
_Comment down the constructor you made_.

**Comment by Sakina-n72** [comment]
hey i already added those

ill do the constructor

**Reply by Sakina-n72** [reply]
i added the constructor also , just check if you can see

**Reply by venkatesh-reddi** [reply]
Yeah sure. Remove the main method as well.

**Reply by venkatesh-reddi** [reply]
Setters and Getters are there, only add empty constructor.

**Comment by Sakina-n72** [comment]
[overalt code 2.txt](https://github.com/user-attachments/files/16679469/overalt.code.2.txt)
 hey im adding the sample for module wise classes so that we can work tomorrow
